skein/core.py

```diff
@@ -298,6 +301,8 @@ class Client(_ClientBase):
         The driver log level. Sets the ``skein.log.level`` system property. One
         of {'ALL', 'TRACE', 'DEBUG', 'INFO', 'WARN', 'ERROR', 'FATAL', 'OFF'}
         (from most to least verbose). Default is 'INFO'.
+    java_opts : str, optional
+        Additional options for the driver
```

Comment:
Slight preference for `java_options` instead of `java_opts` - the abbreviation only saves 3 characters.

Also, the docstring says a string, but the code expects a list of strings. I'm not sure what's best here:

```python
client = skein.Client(..., java_options=("-Dsome.option.one=foo"
                                         "-Dsome.option.two=bar"))
# vs
client = skein.Client(..., java_options=["-Dsome.option.one=foo",
                                         "-Dsome.option.two=bar"])
```

Perhaps accept both - `.append` if a string, otherwise `.extend`?

Also, the comment should clarify that these are *java* options (same elsewhere as well).

```
        Additional java options for the driver.
```

Reply:
I have change the docstring to List[str] as the final object is a list as well.
It seems easier to support only one format instead of str + List[str], right ?
Not sure if one has to put 'list of str' or 'List[str]'. I have taken the newer typings as defined by PEP-484 .